language/Portuguese_Brazil/strings.po

```diff
@@ -3274,3484 +3282,3523 @@ msgstr "[Nenhum]\n"
 msgctxt "input-name"
 msgid "P10 Button 12"
-msgstr ""
+msgstr "Jogador 10 Botão 12"
 
 #: src/emu/inpttype.ipp:483
 msgctxt "input-name"
 msgid "P10 Button 13"
-msgstr ""
+msgstr "Jogador 10 Botão 13"
 
 #: src/emu/inpttype.ipp:484
 msgctxt "input-name"
 msgid "P10 Button 14"
-msgstr ""
+msgstr "Jogador 10 Botão 14"
 
 #: src/emu/inpttype.ipp:485
 msgctxt "input-name"
 msgid "P10 Button 15"
-msgstr ""
+msgstr "Jogador 10 Botão 15"
 
 #: src/emu/inpttype.ipp:486
 msgctxt "input-name"
 msgid "P10 Button 16"
-msgstr ""
+msgstr "Jogador 10 Botão 16"
 
 #: src/emu/inpttype.ipp:487
 msgctxt "input-name"
 msgid "P10 Start"
-msgstr ""
+msgstr "Jogador 10 Start"
 
 #: src/emu/inpttype.ipp:488
 msgctxt "input-name"
 msgid "P10 Select"
-msgstr ""
+msgstr "Selecionar Jogador 10"
 
 #: src/emu/inpttype.ipp:493
 msgctxt "input-name"
 msgid "1 Player Start"
-msgstr ""
+msgstr "Start do Jogador 1"
 
 #: src/emu/inpttype.ipp:494
 msgctxt "input-name"
 msgid "2 Players Start"
-msgstr ""
+msgstr "2 Jogadores Start"
 
 #: src/emu/inpttype.ipp:495
 msgctxt "input-name"
 msgid "3 Players Start"
-msgstr ""
+msgstr "3 Jogadores Start"
 
 #: src/emu/inpttype.ipp:496
 msgctxt "input-name"
 msgid "4 Players Start"
-msgstr ""
+msgstr "4 Jogadores Start"
 
 #: src/emu/inpttype.ipp:497
 msgctxt "input-name"
 msgid "5 Players Start"
-msgstr ""
+msgstr "5 Jogadores Start"
 
 #: src/emu/inpttype.ipp:498
 msgctxt "input-name"
 msgid "6 Players Start"
-msgstr ""
+msgstr "6 Jogadores Start"
 
 #: src/emu/inpttype.ipp:499
 msgctxt "input-name"
 msgid "7 Players Start"
-msgstr ""
+msgstr "7 Jogadores Start"
 
 #: src/emu/inpttype.ipp:500
 msgctxt "input-name"
 msgid "8 Players Start"
-msgstr ""
+msgstr "8 Jogadores Start"
 
 #: src/emu/inpttype.ipp:505
 msgctxt "input-name"
 msgid "Coin 1"
-msgstr ""
+msgstr "Ficha 1"
 
 #: src/emu/inpttype.ipp:506
 msgctxt "input-name"
 msgid "Coin 2"
-msgstr ""
+msgstr "Ficha 2"
 
 #: src/emu/inpttype.ipp:507
 msgctxt "input-name"
 msgid "Coin 3"
-msgstr ""
+msgstr "Ficha 3"
 
 #: src/emu/inpttype.ipp:508
 msgctxt "input-name"
 msgid "Coin 4"
-msgstr ""
+msgstr "Ficha 4"
 
 #: src/emu/inpttype.ipp:509
 msgctxt "input-name"
 msgid "Coin 5"
-msgstr ""
+msgstr "Ficha 5"
 
 #: src/emu/inpttype.ipp:510
 msgctxt "input-name"
 msgid "Coin 6"
-msgstr ""
+msgstr "Ficha 6"
 
 #: src/emu/inpttype.ipp:511
 msgctxt "input-name"
 msgid "Coin 7"
-msgstr ""
+msgstr "Ficha 7"
 
 #: src/emu/inpttype.ipp:512
 msgctxt "input-name"
 msgid "Coin 8"
-msgstr ""
+msgstr "Ficha 8"
 
 #: src/emu/inpttype.ipp:513
 msgctxt "input-name"
 msgid "Coin 9"
-msgstr ""
+msgstr "Ficha 9"
 
 #: src/emu/inpttype.ipp:514
 msgctxt "input-name"
 msgid "Coin 10"
-msgstr ""
+msgstr "Ficha 10"
 
 #: src/emu/inpttype.ipp:515
 msgctxt "input-name"
 msgid "Coin 11"
-msgstr ""
+msgstr "Ficha 11"
 
 #: src/emu/inpttype.ipp:516
 msgctxt "input-name"
 msgid "Coin 12"
-msgstr ""
+msgstr "Ficha 12"
 
 #: src/emu/inpttype.ipp:517
 msgctxt "input-name"
 msgid "Bill 1"
-msgstr ""
+msgstr "Bill 1"
```

Comment:
This is for a banknote reader (or bill accepter in the US), i.e. like a coin slot but for banknotes rather than coins.  Is this the most appropriate translation?

Reply:
Fixed.